posts/inside-rust/2022-05-19-governance-update.md

```diff
@@ -0,0 +1,95 @@
+> * **Reporting progress**: there is a *lot* that happens in the Rust project much of which is not reported on. Having some mechanism for better ensuring that everyone has a good overview of what is happening in the project.
+> * **Active community management**: actively fostering the culture of the Rust project
+> * **Marketing**: the project had previously done more marketing around Rust usage - this work has largely moved to being preformed exclusively by individuals
+> * **Public relations**: The Rust project has obligations to speak to the outside world (i.e., press, companies, educational institutions, other OSS projects, etc.)
+> * **User outreach**: while PR is a push mechanism, the project also needs some sort of pull mechanism for engaging with users and understanding their needs rather than solely relying on the individual insights that contributors bring.
+> * **Vision work**: establishing high level project wide goals that cross between team boundaries.
+> 
+> ## Failure modes
+> 
+> The following are a list of identified failure modes that we want to ensure we avoid when discussing a new governance structure:
+> 
+> * **Lack of resources for admin/project management work**: Administrative and project management work is typically less resourced than technical work. Volunteers are typically more drawn towards technical work, and companies tend to fund technical work since they will more easily see a return on that investment. A properly running project is viewed as “table stakes” and thus less likely to see investment without purposeful resource allocation. A failure mode would be not having admin/project management functions be funded and eventually withering away leaving that work not done or done by those who are already busy with other concerns.
+> * **Project leadership not held accountable**: Many teams depend on the work of other teams for their own success. When a team feels another team is not delivering on what they need to succeed, this can lead to cascading failures or even outright conflict. Holding others accountable can be difficult because it requires clear expectations, channels for delivering clear and actionable feedback, consequences for consistently not living up to expectations, and mechanisms for handling conflict should it arise.
+> * **Project leadership detached from project**: As the project grows in complexity so does that admin/project management overhead. It is possible for a project wide decision making body to lose touch with goings on in the project as they become busy due to this overhead. There are two ways this can manifest itself: the leadership body fails to keep up with what’s happening in the project and/or the project members lose sight of the leadership body leading to degraded authority. A failure mode would be that the project leadership body becomes detached from the project and the two effectively start acting independently.
```

Comment:
```suggestion
> * **Project leadership detached from project**: As the project grows in complexity so does that admin/project management overhead. It is possible for a project wide decision making body to lose touch with goings-on in the project as they become busy due to this overhead. There are two ways this can manifest itself: the leadership body fails to keep up with what’s happening in the project and/or the project members lose sight of the leadership body leading to degraded authority. A failure mode would be that the project leadership body becomes detached from the project and the two effectively start acting independently.
```